src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -75,7 +75,8 @@ uct_rc_mlx5_ep_zcopy_post(uct_rc_mlx5_ep_t *ep,
                                    UCT_IB_MAX_ZCOPY_LOG_SGE(&iface->super.super));
 
     uct_rc_txqp_add_send_comp(&iface->super, &ep->super.txqp, handler, comp, sn,
-                              UCT_RC_IFACE_SEND_OP_FLAG_ZCOPY);
+                              UCT_RC_IFACE_SEND_OP_FLAG_ZCOPY,
+                              uct_iov_total_length(iov, iovcnt));
```

Comment:
is this done also for put zcopy?

Reply:
yes, and others. Better avoid? We would need to create a separate func for gets then

Reply:
better avoid, let's try to pass length to this func from callers?

Reply:
BTW. do we have completed length in send CQE maybe?